packages/api/src/hooks/internal/WebChatAPIContext.ts

```diff
@@ -45,17 +53,21 @@ type WebChatAPIContext = {
     linkOptions: { externalLinkAlt: string }
   ) => string;
   scrollToEndButtonRenderer?: ScrollToEndButtonComponentFactory;
-  selectVoice?: (voices: (typeof window.SpeechSynthesisVoice)[], activity: WebChatActivity) => void;
+  selectVoice?: (
+    voices: (typeof window.SpeechSynthesisVoice)[],
+    activity: WebChatActivity
+  ) => typeof window.SpeechSynthesisVoice;
   sendEvent?: (name: string, value: any) => void;
-  sendFiles?: (files: File[]) => void;
```

Comment:
Previously, `(files: File[]) => void` was wrong, it should be `(files: WebChatAttachment[]) => void`.